index.html

```diff
@@ -16,42 +16,246 @@
             <img src="images/humbger.png" alt="toggle menu" />
           </div>
         </div>
-      </nav></header>
-      <section class="headline container">
-        <div class="heading">
-          <h1>I’m Glory <br>Glad to see<br> you!</h1>
+      </nav>
+    </header>
+    <section class="headline container">
+      <div class="heading">
+        <h1>
+          I’m Glory <br />Glad to see<br />
+          you!
+        </h1>
+      </div>
+      <div class="paragraph-one">
+        <p class="paragraph">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="lets-connect"><a href="#">LET’S CONNECT</a></p>
+      </div>
+      <ul class="social-media-links">
+        <li>
+          <a href="#"><img src="images/twitter.png" alt="twitter" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Linkedinicon.png" alt="linkedin" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector.png" alt="microverse" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/gitgub.png" alt="github" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="images/Vector3.png" alt="vector" /></a>
+        </li>
+      </ul>
+    </section>
+    <main>
+      <section class="grid-container works-section container">
+        <div>
+          <div class="grid-item">
+            <div class="card">
+              <div class="project">
+                <img src="/images/first-project.png" alt="my first project" />
```

Comment:
Good job so far 👍. Kindly check your image path, it seems it's broken and the image is not displaying. Also, kindly remove unnecessary spaces and let your code be in line with the Figma design 👇
<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/205440689-ef5af6d4-c858-4df9-8dc9-1c79e82f3b15.png">